### DISCUSSION TITLE
Is there any plan for supporting code highlighting?

### DISCUSSION THREAD

**Discussion by kshired** [markdoc/markdoc#36] [Q&A] [answered]
Hi markdoc!

In these days, there are a lot of markdown editor support code highlighting. But it seems that this project doesn't support code highlighting.

So I wonder if u have a plan for supporting code highlighting? 

Thank u.

**Comment by kshired** [comment, marked as answer]
Oh, I found [example](https://markdoc.io/docs/examples#syntax-highlighting) using prism. 



But, I still wonder u have a plan for "official support" or not.

**Reply by mfix-stripe** [reply]
@kshired there are many different ways do do code highlighting, so we hope folks create (and contribute their own packages for) their own preferred method of code highlighting (e.g. Prism, CodeMirror, etc.)

For example, we imagine someone could create a `markdoc-prism-node` package that simply contains the components you saw in [this example](https://markdoc.io/docs/examples#syntax-highlighting). But if someone else wants to use a different system, they could just create another small package for that use case (or simply copy-paste these small bits into their app).
